Resources/Prototypes/Corvax/Entities/Clothing/Neck/pins.yml

```
@@ -1,3 +1,19 @@
+- type: entity
+  parent: ClothingNeckPinBase
+  id: ClothingNeckAtharaPin
+  name: starry sky pin
+  description: The pin of starry sky.
+  components:
+  - type: Sprite
+    sprite: Corvax/Clothing/Neck/Misc/pins.rsi
+    layers:
+    - state: ath
```

```suggestion
  - type: Sprite
    sprite: Corvax/Clothing/Neck/Misc/pins.rsi
    state: ath
```
и так во всех в этом файле сделай